pkg/agent/statusupdate.go

```diff
@@ -163,7 +164,3 @@ func runStatusUpdateCheck() error {
 func UpdateLocalActivityTime() {
```

Comment:
Are we even using the currentActivityTime now? Is this function needed?

Reply:
You're right...will remove